configuration/pih/htmlforms/hiv/zl/hiv-dispensing.xml

```diff
@@ -120,10 +120,21 @@
 
         var currentEncounterDate = new Date();
         var encounterDate = '<lookup expression="encounter.getEncounterDatetime().getTime()"/>';
-        if (typeof encounterDate !== "undefined" &amp;&amp; encounterDate !== null &amp;&amp; (encounterDate.length > 0)) {
-          currentEncounterDate = new Date(+encounterDate);
+          if (typeof encounterDate !== "undefined" &amp;&amp; encounterDate !== null &amp;&amp; (encounterDate.length > 0)) {
+            currentEncounterDate = new Date(+encounterDate);
+        } else {
+          // look for the encounterDate datepicker widget
+          var encounterDateValue = jq("#encounterDate .hasDatepicker");
```

Comment:
Does the getField method not work here?  (Not a big deal if it doesn't, just curious)

Reply:
Thanks @mogoodrich ! Yes, I have noticed in the javascript console that sometimes when the form loads it throws a null getField() method not found. I have tried to debug it, but when I set a breakpoint it works b/c there is plenty time for the page and all depending js scripts to load. When I switched to using jquery selector I did not see that error anymore.

Reply:
Interesting... something to keep an eye on for other places where it may cause issues.